timApp/velp/velpgroups.py

```diff
@@ -51,6 +49,56 @@ class CreatedVelpGroup:
     created_new_group: bool = True
 
 
+@dataclass
+class GroupSelection:
+    """Stores information on a velp group's selection status"""
+
+    id: int
+    selected: bool
+
+    def to_json(self) -> dict[str, int | bool]:
+        return {"id": self.id, "selected": self.selected}
+
+
+@dataclass
+class VelpGroupSelectionInfo:
+    """Stores information on selected and default velp groups for a document. Used for
+    passing velp group selection data to the user interface.
+    """
+
+    target_ids: list[str]
+    selections: list[list[GroupSelection]]
+
+    def append(self, target_id: str, gs: GroupSelection) -> None:
+        if len(target_id) == 0:
```

Comment:
Vähän "pythonimaisempi" tapa tarkistaa tyhjä jono:

```suggestion
        if not target_id:
```